src/main/webapp/app/pages/companyPage/CompanyAdditionalInfo.tsx

```diff
@@ -0,0 +1,142 @@
+import React from 'react';
+import FormSelectWithLabelField from 'app/shared/select/FormSelectWithLabelField';
+import FormInputField from 'app/shared/input/FormInputField';
+import { CompanyAdditionalInfoDTO } from 'app/shared/api/generated/API';
+import { FormTextAreaField } from 'app/shared/textarea/FormTextAreaField';
+
+const defaultAdditionalInfo: CompanyAdditionalInfoDTO = {
```

Comment:
Having a default means we will always have companyAdditionalInfo populated in the DB correct? Historically we don't add the default but leave the filed empty in the DB.

Reply:
Yes, but shouldn't we always have it populated going forward?

When you say empty do you mean that you don't include the field at all in the json?

Reply:
Yeah, if the user does not have any additional info, the DB has no content.